sox.features.js

```diff
@@ -1,27 +1,33 @@
+/* globals Notifier, fkey, jQuery */
 (function(sox, $) {
   'use strict';
 
   sox.features = { //SOX functions must go in here
     moveBounty: function() {
       // Description: For moving bounty to the top
 
-      $('.bounty-notification').insertAfter('.question .fw-wrap');
-      $('.bounty-link.bounty').closest('ul').insertAfter('.question .fw-wrap');
+      const bountyLink = document.querySelector('.bounty-link.bounty');
+      if (!bountyLink) return; // question not eliglible for bounty
+      const bountyLinkDiv = bountyLink.parentElement.parentElement;
+      const cloneBounty = bountyLinkDiv.cloneNode(true);
+      bountyLinkDiv.previousElementSibling.insertAdjacentElement('beforebegin', cloneBounty);
+      bountyLinkDiv.remove();
     },
 
     dragBounty: function() {
       // Description: Makes the bounty window draggable
 
-      const target = document.getElementById('question');
-      sox.helpers.observe(target, '#start-bounty-popup', () => {
-        $('#start-bounty-popup').draggable({
+      sox.helpers.addAjaxListener('\\/posts\\/bounty\\/\\d+', () => {
+        const bountyPopup = document.querySelector('#start-bounty-popup');
```

Comment:
I might be wrong but I believe `getElementById` is supposed to be more efficient than `querySelector` when targeting an ID.

Could you possibly do a find/replace with something like `document.querySelector('#[^\s]')` and replace with the `getElementById` version without the `#`?

Reply:
I used `.querySelector(All)` mostly for consistency. Do you have any reference for the fact that `.getElementById` is faster, though? If so, I'd be happy to change it!

Reply:
I just ran a benchmark to see and it does look like there's a slight difference: https://www.measurethat.net/Benchmarks/Show/2488/0/getelementbyid-vs-queryselector

However, thank you so much for your work on this PR -- I'm going to merge it in now as this is an awesome improvement already!

Reply:
I notice you've marked it as draft -- would you be happy for this to be merged in now? :)